code/game/objects/items/weaponry.dm

```diff
@@ -20,10 +20,11 @@
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == "head")
 		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");

```

Comment:
>Bangammer

How did everyone miss this

Reply:
It's deliberate

Reply:
https://github.com/tgstation/tgstation/pull/30173